omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/visit/ParserEncounterIntoSimpleObjectsTest.java

```diff
@@ -296,6 +300,51 @@ public void testParsingSimpleObs() throws Exception {
 		assertThat(path(parsed.getObs(), 0, "question"), is((Object) consultNote));
 		assertThat(path(parsed.getObs(), 0, "answer"), is((Object) valueText));
 	}
+	@Test
+	public void testParsingNumericObsShouldHandleAllowDecimal() throws Exception {
+		ConceptDatatype numericDatatype  = new ConceptDatatype() ;
```

Comment:
These two tests pass even without your above fix. I thought they would start by failing and then your changes above make them pass. Correct?

Reply:
Yes, i made some adjustments